packages/@groupby/elements-sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -136,7 +140,12 @@ export default class SaytDriverPlugin<P = Record> implements Plugin {
    * @param event Event that contains the Search API request payload.
    */
   fetchProductData(event: CustomEvent<SaytProductsRequestPayload>): void {
```

Comment:
The documentation for this function says that it 

> dispatches events on success and failure. 

Is it worth expanding on that now to define what events those are since they aren't just an error or success event?

Reply:
Turns out the search driver version of this already does this, so it should be updated here too.